BEASTIE/beastie_step1.py

```diff
@@ -2,21 +2,18 @@
 #=========================================================================
 # 2021 Xue Zou (xue.zou@duke.edu)
 #=========================================================================
-import logging
 import os
 import sys
-
+import logging
 import pandas as pd
-
+from pathlib import Path
```

Comment:
this should be moved down a line -- all the "import"s should be together and all the "from"s should be together.